### PR TITLE
Scala2.13.0

### PR DESCRIPTION
**Pull request by raboof**
Depends on:
* [x] https://github.com/scalapb/ScalaPB/issues/593
* [x] #598

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -65,6 +67,7 @@ lazy val mavenPlugin = Project(
   .settings(Seq(
     publishMavenStyle := true,
     crossPaths := false,
+    crossScalaVersions := Seq(scala212),
```
Comment (johanandren) [inline build.sbt]: Different style here vs above with removing 2.13 vs replacing with a list with only 2.12

File: build.sbt [hunk 2]
```diff
@@ -162,7 +166,8 @@ lazy val docs = Project(
     ),
     resolvers += Resolver.jcenterRepo,
     publishRsyncArtifact := makeSite.value -> "www/",
-    publishRsyncHost := "akkarepo@gustav.akka.io"
+    publishRsyncHost := "akkarepo@gustav.akka.io",
+    crossScalaVersions := List(scala212, scala213),
```
Comment (johanandren) [inline build.sbt]: docs need cross versions?

File: plugin-tester-java/build.gradle [hunk 3]
```diff
@@ -27,7 +27,7 @@ akkaGrpc {
   generateClient = true
   generateServer = true
   serverPowerApis = true // This isn't default
-  extraGenerators = [ 'akka.grpc.gen.scaladsl.play.PlayScalaClientCodeGenerator', 'akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator' ]
+  extraGenerators = [ ]
```
Comment (johanandren) [inline plugin-tester-java/build.gradle]: How is this related to this PR? Does it also remove play modules, is it on top of the remove play modules-pr?

Comment (marcospereira) [reply plugin-tester-java/build.gradle]: Apparently, this is built on top of #598. So maybe @raboof need to do a rebase here?

Comment (raboof) [reply plugin-tester-java/build.gradle]: (:+1: rebased for smaller diff)

File: build.sbt (hunk 2 above)
Comment (raboof) [reply build.sbt]: I think I did this because docs depends on cross-built modules, and in that case sbt requires the 'depending' module to also be cross-built (https://github.com/sbt/sbt/issues/3465)

File: build.sbt (hunk 1 above)
Comment (raboof) [reply build.sbt]: good point, changing to `:= Seq(scala212)` for all of those since they're only used at the 'plugin side'

File: project/Dependencies.scala [hunk 4]
```diff
@@ -7,15 +7,16 @@ object Dependencies {
...
+    val scala213 = "2.13.0"
 
-    val akka = "2.5.20"
+    val akka = "2.5.23"
     val akkaHttp = "10.1.8"
 
     val grpc = "1.20.0" // checked synced by GrpcVersionSyncCheckPlugin
     val config = "1.3.3"
-    val sslConfig = "0.3.6"
+    val sslConfig = "0.4.0"
```
Comment (marcospereira) [inline project/Dependencies.scala]: Maybe you want to use 0.3.8.
